glide-core/src/request_type.rs

```diff
@@ -341,6 +343,8 @@ impl From<::protobuf::EnumOrUnknown<ProtobufRequestType>> for RequestType {
             ProtobufRequestType::XDel => RequestType::XDel,
             ProtobufRequestType::LMove => RequestType::LMove,
             ProtobufRequestType::BLMove => RequestType::BLMove,
+            ProtobufRequestType::BitField => RequestType::BitField,
+            ProtobufRequestType::BitFieldReadOnly => RequestType::BitFieldReadOnly,
```

Comment:
```suggestion
            ProtobufRequestType:: BitField_Ro => RequestType:: BitField_Ro,
```

Reply:
@acarbonetto, why?

Reply:
The command is: `bitfield_ro`
We don't replace letters, for example: `XDel`.  Not `XDelete` or `StreamDelete`. 
What's different about the `_ro`?